protocol.go

```diff
@@ -12,7 +12,7 @@ import (
 type ProtocolVersion uint64
 
 // ProtocolVersion type stores a Tarantool protocol feature.
-type ProtocolFeature uint64
+type ProtocolFeature iproto.Feature
```

Comment:
Nit: we could replace the `ProtocolFeature` type by the `iproto.Feature`.

Reply:
Made this change in a different commit.